plugins/grype/surfactantplugin_grype.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024 Lawrence Livermore National Security, LLC
+# See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+import gzip
+import subprocess
+import tempfile
+from typing import List, Optional
+
+from loguru import logger
+
+import surfactant.plugin
+from surfactant.sbomtypes import SBOM, Software
+
+
+def check_if_grype_installed() -> bool:
+    try:
+        result = subprocess.run(["grype", "--help"], capture_output=True, check=False).returncode
+    except FileNotFoundError:
+        result = 1
+    if result != 0:
+        logger.warning("Install grype for the grype plugin to run")
+    return result == 0
+
+
+disable_plugin = not check_if_grype_installed()
+
+
+def run_grype(filename: str) -> object:
+    result = subprocess.run(["grype", filename], capture_output=True, check=False)
+    if result.returncode != 0:
+        logger.warning(f"Running grype on {filename} failed")
+        return None
+    output = result.stdout.decode()
+    to_ret = []
+    for line in output.split("\n"):
+        columns = line.split()
+        # Skip empty lines and the header
+        if len(columns) == 0 or columns[0] == "NAME":
```

Comment:
```suggestion
    # skip the header on the first line
    for line in output.split("\n")[1:]:
        columns = [s.strip() for s in line.split('  ') if s.strip()]
        # Skip empty lines
        if len(columns) == 0:
```

I noticed that there was a bug resulting in an extra column when FIXED-IN is `(won't fix)` due to the space. It looks like grype uses [padding of 2 spaces](https://github.com/anchore/grype/blob/10e9f4317e28059066d56eef84f1eded812c1523/grype/presenter/table/presenter.go#L99) between columns, so splitting on two spaces (and then removing the empty fields) seems to fix the issue (and potentially a similar issue if a version number or package name ever has a space in it).